cmd/tailscale/cli/cli_test.go

```diff
@@ -480,6 +480,19 @@ func TestCheckForAccidentalSettingReverts(t *testing.T) {
 			distro: "", // not Synology
 			want:   accidentalUpPrefix + " --hostname=foo --accept-routes",
 		},
+		{
+			name:  "profile_name_ignored_in_up",
+			flags: []string{"--hostname=foo"},
```

Comment:
Should this be `--profile-name`?

Reply:
This checks that even though `--profile-name` arg is not defined AND we already have a `ProfileName` set it doesn't result in the accidental revert message.